src/klein/test/test_headers.py

```diff
@@ -33,25 +33,14 @@
 
 def encodeName(name):
     # type: (Text) -> Optional[bytes]
-    try:
-        return name.encode(HEADER_NAME_ENCODING)
-    except UnicodeEncodeError:
-        return None
+    return name.encode(HEADER_NAME_ENCODING)
 
 
 def decodeName(name):
     # type: (bytes) -> Text
     return name.decode(HEADER_NAME_ENCODING)
 
 
-def encodeValue(name):
```

Comment:
Looks like [`encodeName` is used where I expected `encodeValue`](https://github.com/twisted/klein/blob/e5a66722db4d5fcc24975caef761f9f41dce291b/src/klein/test/test_headers.py#L107).  Maybe these functions can be given a more generic name, like `encodeHeaderElement` and `decodeHeaderElement`?

Reply:
Oh, no, this is a real bug. Will fix.